troposphere/static/js/components/common/breadcrumb/Breadcrumb.react.js

```diff
@@ -23,13 +23,10 @@ define(function(require) {
 
     renderLink: function() {
       var class_names = this.props.breadcrumb.state;
-      var divStyle = {
-        width: this.props.width + '%'
-      };
 
       return (
-          <div style={divStyle} onMouseOver={this.mouseOver} onMouseOut={this.mouseOut} className={class_names} onClick={this.crumbClicked}>
-            {this.props.breadcrumbText}
+          <div onMouseOver={this.mouseOver} onMouseOut={this.mouseOut} className={class_names} onClick={this.crumbClicked}>
+            <span className="hidden-xs">Step </span> {this.props.breadcrumbText}
```

Comment:
If we're going to go this far, perhaps we should replace "Step ##" with an actual 'informative' title, and show that when the screen is large enough to bare it?